### PR TITLE
temporary fps boost

### PR DESCRIPTION
**Pull request by DaddyWesker**
This is temporary. Using this stub solution fps in fullscreen on my laptop is now 36-38 instead of 22-24. But it is better to find better solution.

### CODE REVIEW

File: src/main/java/io/singularitynet/Client/VideoHook.java [hunk 1]
```diff
@@ -301,33 +303,40 @@ public void postRender(WorldRenderContext event)
...
-                int jo_len = jo_bytes.length;
-                this.buffer = this.videoProducer.getFrame(this.missionInit);
-                if (this.buffer != null)
-                {
-                    ByteBuffer jo_len_buffer = ByteBuffer.allocate(4).order(ByteOrder.BIG_ENDIAN).putInt(jo_len);
-                    jo_len_buffer.flip();
-                    int frame_buf_len = this.buffer.capacity();
-                    this.buffer.limit(frame_buf_len);
-                    ByteBuffer[] buffers = {jo_len_buffer, ByteBuffer.wrap(jo_bytes), this.buffer};
+                if (tictac.equals("tic")) {
```
Comment (noskill) [inline src/main/java/io/singularitynet/Client/VideoHook.java]: we can move tictac check at line 302:

if (AddressHelper.getMissionControlPort() == 0  || tictac.equals("tac"))

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/Client/VideoHook.java]: @noskill Done.

File: src/main/java/io/singularitynet/Client/VideoHook.java [hunk 2]
```diff
@@ -297,10 +299,14 @@ public void postRender(WorldRenderContext event)
...
         try
         {
-            if (AddressHelper.getMissionControlPort() == 0) {
+            if (AddressHelper.getMissionControlPort() == 0 || tictac.equals("tac")) {
+                tictac = "tic";
                 success = true;
                 time_after_render_ns = System.nanoTime();
-            } else {
+            }
+            else {
```
Comment (noskill) [inline src/main/java/io/singularitynet/Client/VideoHook.java]:
```suggestion
            } else {
```

File: src/main/java/io/singularitynet/Client/VideoHook.java [hunk 3]
```diff
@@ -297,10 +299,13 @@ public void postRender(WorldRenderContext event)
...
         try
         {
-            if (AddressHelper.getMissionControlPort() == 0) {
+            if (AddressHelper.getMissionControlPort() == 0 || tictac.equals("tac")) {
+                tictac = "tic";
                 success = true;
                 time_after_render_ns = System.nanoTime();
             } else {
+                tictac = "tac";
+                this.buffer.clear();
```
Comment (noskill) [inline src/main/java/io/singularitynet/Client/VideoHook.java]: it can fail if buffer == null

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/Client/VideoHook.java]: Oops, I've missed this line. It is from my experiments. I've deleted it.

**Review by noskill — APPROVED** [review]

**Comment by noskill** [comment]
@DaddyWesker there is error during test:
TypeError: RobustObserver.getBlockFromBigGrid() takes 1 positional argument but 2 were given

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**